addons/l10n_it_edi/models/res_config_settings.py

```diff
@@ -70,6 +70,7 @@ def _set_l10n_it_edi_sdicoop_register_demo_mode(self):
 
             real_proxy_users = self.env['account_edi_proxy_client.user'].sudo().search([
                 ('company_id', '=', config.company_id.id),
+                ('proxy_type', '=', 'l10n_it_edi'),
```

Comment:
thanks! I wonder if we should add it on line 97 just in case :thinking:

Reply:
Yes this make sense